### PR TITLE
.github/ISSUE_TEMPLATE: various formatting and typo fixes

### PR DESCRIPTION
**Pull request by pietrushnic**
This patch wrap at 80 characters to improve readability and review
process. It fixes typo and make templates more consistent by using the
same wording.

Signed-off-by: Piotr Król <piotr.krol@3mdeb.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 name: Bug report
-about: 'Report a problem in existing functionality, including documentation and infrastructure.'
+about: Report a problem in existing functionality, including documentation and infrastructure.
```
Comment (dpsmith) [inline .github/ISSUE_TEMPLATE/bug-report.md:3]: I haven't looked at the other templates so it may be like this in them but why did you remove the quotes here but left them on the title, labels, and assignees fields?

Comment (pietrushnic) [reply .github/ISSUE_TEMPLATE/bug-report.md:3]: @dpsmith I just tried to be consistent across about sections for all templates, if we want to leave `'` here then let's add those wherever those are missing. Please let me know what is your policy and I will correct it accordingly. TBH I'm not sure if it will cause any rendering issues.

Comment (dpsmith) [reply .github/ISSUE_TEMPLATE/bug-report.md:3]: No worries, at that point had not looked across the other templates. Honestly I don't think it matters and agree it likely does not affect rendering. I was more curious why some fields get quotes and others don't, which might be a question better for @rossphilipson. It might be as simple as an artifact from the templates that were borrowed from.

Comment (rossphilipson) [reply .github/ISSUE_TEMPLATE/bug-report.md:3]: Ok I am going to just take this to make everything consistent. If there are rendering issues I will PR a consistent use of using single quotes.

**Review by rossphilipson — APPROVED** [review]
This looks good to me. I am going to merge it and see how things look.

Reviewed-by: Ross Philipson <ross.philipson@oracle.com>
